MIGRATION.md

```diff
@@ -105,6 +106,68 @@
 
 ## From version 5.3.x to 6.0.x
 
+### Backgrounds addon has a new api
+
+Starting in 6.0, the backgrounds addon now receives an object instead of an array as parameter, with a property to define the default background.
+
+Consider the following example of its usage in both `preview.js` and `Button.stories.js`:
+```jsx
+// preview.js
+import { addParameters } from '@storybook/react'; // <- or your storybook framework
```

Comment:
Did you know in 6.0 you can just do this:

```js
export const parameters = {
  backgrounds: [
    { name: 'twitter', value: '#00aced', default: true },
    { name: 'facebook', value: '#3b5998' },
  ],
};
```
Shorter, and avoids framework-specific code!

Reply:
Is that new in CSF?

Reply:
New in `preview.js` configuration (thanks to @tmeasday)

Reply:
I see.

Reply:
That's awesome! Although now I'm confused. Do you want me to update the docs or you just wanted to tell me about this nice feature? 😂

Reply:
Could be moved to main.js eventually.

Reply:
done!

Reply:
> Could be moved to main.js eventually.

Yep, definitely on the horizon I reckon. You can also export `decorators` and `argTypeEnhancers`, which maybe make less sense in `main.js`.

Reply:
@ndelangen if you wanted to add a parameters field to `main.js` I would be down with it.

Although we might have to resolve this one: https://github.com/storybookjs/storybook/issues/10005